apps/api/src/app/auth/auth.controller.ts

```diff
@@ -114,6 +114,11 @@ export class AuthController {
       url += `&configurationId=${configurationId}`;
     }
 
+    const invitationToken = JSON.parse(request.query.state).invitationToken;
+    if (invitationToken) {
+      url += `&invitationToken=${invitationToken}`;
+    }
```

Comment:
On the invitation page, we do show forms "Sign In/Up & Accept" with a GitHub button, when that button is clicked we redirect the user to GH OAuth, and in query params we do pass the `invitationToken`. When a user is successfully logged in he will be redirected to the login page, where we do read that `invitationToken` from the query params and trigger the "auto-accept invite" logic.